index.js

```diff
@@ -38,6 +42,95 @@ function filterContentSources(content, stream) {
     }
 }
 
+function filterMsidFromRecvonlySources(description) {
+    description.contents.forEach(function(content) {
+        content.application.sources.forEach(function(source) {
+            source.parameters = source.parameters.filter(function(param) {
+                if (param.key === 'msid' && (description.senders ==='initiator' || description.senders ==='none')) {
+                    return false;
+                }
+                return true;
+            });
+        });
+    });
+
+    return description;
+}
+
+function getContent(content) {
```

Comment:
nit: maybe give this a more descriptive name like `getRtpContentsWithSources`

Reply:
renamed :)